### PR TITLE
feat(components/text-editor): tokenize text editor styles

### PR DESCRIPTION
**Pull request by Blackbaud-ErikaMcVey**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2936/storybook/)

Component Storybooks:

- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2936/storybooks/colorpicker-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2936/storybooks/pages-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2936/storybooks/text-editor-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2936/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2936/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2936/playground/)

File: libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss [hunk 1]
```diff
@@ -1,64 +1,238 @@
...
+        var(--sky-color-border-input-base)
+      )
+  );
+
   &.sky-text-editor-disabled {
-    .sky-toolbar *,
-    .sky-text-editor-wrapper * {
-      color: var(--sky-text-color-deemphasized) !important;
-    }
+    background-color: var(--sky-color-background-input-disabled);
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss:74]: Looks like the spec calls for tokens on the border here that I don't see. I don't see a functional difference when I tried it - but probably should still have the token.

File: libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss [hunk 2]
```diff
@@ -1,64 +1,238 @@
...
+
+      margin-bottom: var(--sky-override-text-editor-label-margin-bottom, 0);
+    }
   }
 
   .sky-text-editor-wrapper {
     display: flex;
     flex-wrap: wrap;
-    background-color: white;
+    background-color: var(
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss:127]: The background color is set here - but I don't see it set on the toolbar or label portion.

File: libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss [hunk 3]
```diff
@@ -1,64 +1,238 @@
...
+      --sky-override-text-editor-background-color,
+      var(--sky-color-background-input-base)
+    );
     width: 100%;
     height: 300px;
```
Comment (Blackbaud-TrevorBurch) [inline libs/components/text-editor/src/lib/modules/text-editor/text-editor.component.scss:132]: Nothing to change - just going to call out that I wish this wasn't a magic number. Nothing you did - just an observation.

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Pull request merged**